### ISSUE TITLE
Error updating

### ISSUE THREAD

**Issue by Huseyn050** [ddvk/remarkable-update#11]
Followed all the instructions but after showing updating to 2.8 for a few seconds it says error updating,,, (I'm trying to go from 2.9 to 2.8) Stuck on what to do here. Rebooting afterwards with and without running switch.sh does nothing. Need help

Heres my log from terminal which can hopefully clear something up:
requested:  2.9.1.217
platform:  reMarkable2
Response:
<?xml version="1.0" encoding="UTF-8"?>
 <response protocol="3.0" server="prod">
  <daystart elapsed_seconds="37145" elapsed_days="5179"/>
  <app status="ok" appid="{98DA7DF2-4E3E-4744-9DE6-EC931886ABAB}">
    <event status="ok"/>
    <updatecheck status="ok">
      <urls>
        <url codebase="http://{192.168.0.114}:{8000}/"/>
      </urls>
      <manifest version="2.8.0.86">
        <packages>
          <package required="true" hash="lmNchpajUTMAOwqDMorSDje7mGU=" name="updates/2.8.0.86_reMarkable2.signed" size="68377342"/>
        </packages>
        <actions>
          <action successsaction="default" sha256="jQTFkPJrYgSYpKgjeKg4IHqvf+18WWjzMJ3uc2LdBxQ=" event="postinstall" DisablePayloadBackoff="true"/>
        </actions>
      </manifest>
    </updatecheck>
    <ping status="ok"/>
  </app>
</response>

192.168.0.105 - - [21/Aug/2021 02:00:18] "POST / HTTP/1.1" 200 -
<?xml version="1.0" encoding="UTF-8"?>
<request protocol="3.0" version="2.9.1.217" requestid="{735481ee-6dc6-460b-955a-33473bd324e1}" sessionid="{67413992-7417-48c4-89d3-0aad3751033d}" updaterversion="0.4.2" installsource="ondemandupdate" ismachine="1">
    <os version="codex " platform="reMarkable2" sp="2.9.1.217_armv7l" arch="armv7l"></os>
    <app appid="{98DA7DF2-4E3E-4744-9DE6-EC931886ABAB}" version="2.9.1.217" track="Prod" ap="Prod" bootid="{797b7b46-672f-4afe-adac-5240fed9ac68}" oem="RM110-130-34860" oemversion="" alephversion="2.9.1.217" machineid="414c14b21c7c4a8c8f5e468f546c08ed" lang="en-US" board="" hardware_class="" delta_okay="false" nextversion="2.8.0.86" brand="" client="" >
        <event eventtype="13" eventresult="1"></event>
    </app>
</request>

Update done
<?xml version="1.0" encoding="UTF-8"?>
<request protocol="3.0" version="2.9.1.217" requestid="{e0ce7e51-4aeb-4021-8a94-bc3b3ff9d480}" sessionid="{67413992-7417-48c4-89d3-0aad3751033d}" updaterversion="0.4.2" installsource="ondemandupdate" ismachine="1">
    <os version="codex " platform="reMarkable2" sp="2.9.1.217_armv7l" arch="armv7l"></os>
    <app appid="{98DA7DF2-4E3E-4744-9DE6-EC931886ABAB}" version="2.9.1.217" track="Prod" ap="Prod" bootid="{797b7b46-672f-4afe-adac-5240fed9ac68}" oem="RM110-130-34860" oemversion="" alephversion="2.9.1.217" machineid="414c14b21c7c4a8c8f5e468f546c08ed" lang="en-US" board="" hardware_class="" delta_okay="false" nextversion="2.8.0.86" brand="" client="" >
        <event eventtype="3" eventresult="0" errorcode="268435465"></event>
    </app>
</request>

**Comment by wbernoudy** [comment]
Hey Huseyn050, I have no contributed to this project, but I was having the same issue until I told the host machine to use the hostname I was using on the remarkable tablet. I had set
```
SERVER=http://MYHOSTNAME:8000
```
in `/usr/share/remarkable/update.conf` on the remarkable, so I had to run

```
python serve.py MYHOSTNAME
```
on my computer. Hope that helps.

**Comment by Huseyn050** [comment]
@wbernoudy you mean putting the same IP on both? like the local IP address? cause thats what I did and it refuses to work. Unless you meant something else

**Comment by wbernoudy** [comment]
Yes, that is what I meant. Perhaps you can post the entire output of `python serve.py`, and of `journalctl -u update-engine` on the remarkable?

**Comment by Huseyn050** [comment]
Hi again! sorry for the late reply, I attached the full output for python serve.py but unfortunately I couldn't get the log from the remarkable for some reason. Might try again later. Please tell me if you notice anything specifically, thanks!
[Remarkable Attempted Downgrade log.txt](https://github.com/ddvk/remarkable-update/files/7145243/Remarkable.Attempted.Downgrade.log.txt)

**Comment by wbernoudy** [comment]
When I run `python serve.py HOSTNAME`, the logs start with
```
Using hostname, to override use: ./serve.py hostname
Device should use:  http://HOSTNAME:8000/
Available updates: {'reMarkable2': ('2.7.0.51', '2.7.0.51_reMarkable2.signed')}
Starting fake updater: 8000
```
which seems to be missing from your logs.

It looks like your server is using `http://{192.168.0.114}:{8000}` and you set your remarkable to use `http://Huseyns-MacBook-Pro.local:8000`. If it's still not working, try to post the exact command you used to run the server, the entire logs, the contents of `/usr/share/remarkable/update.conf`, and all the logs from the remarkable again.
